cohortextractor/validate_dummy_data.py

```diff
@@ -124,15 +124,14 @@ def date_validator(value):
         "exists": bool_validator,
         "count": int,
     }
-
-    if (
-        hasattr(query_node, "column")
-        and query_node.column in columns_to_validator_mapping
-    ):
-        return columns_to_validator_mapping[query_node.column]
     if (
         hasattr(query_node, "function")
         and query_node.function in functions_to_validator_mapping
     ):
         return functions_to_validator_mapping[query_node.function]
+    if (
+        hasattr(query_node, "column")
+        and query_node.column in columns_to_validator_mapping
+    ):
+        return columns_to_validator_mapping[query_node.column]
```

Comment:
A query node always has a column; if we check for column first, it uses the validator for the column rather than the one for the function.  An `exists` function should have a boolean validator, for example

Reply:
Oh, so you found an actual bug by increasing code coverage? That's great.